### PR TITLE
add finetune single_class_dataset

### PR DESCRIPTION
**Pull request by yuedongli1**
Thank you for your contribution to the MindYOLO repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindyolo/blob/master/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: examples/finetune_car_detection/rename.py [hunk 1]
```diff
@@ -5,18 +5,17 @@
 start_num = 1
 
 # 获取当前目录下所有的.txt文件
-files = [f for f in os.listdir('images/train') if f.endswith(extension)]
+files = [f for f in os.listdir('images/val') if f.endswith(extension)]
```
Comment (CaitinZhao) [inline examples/finetune_car_detection/rename.py:8]: 为啥要换这个

Comment (yuedongli1) [reply examples/finetune_car_detection/rename.py:8]: 只需要对验证集做改名和生成json文件，这里怕误导用户

File: examples/finetune_car_detection/crejson.py [hunk 2]
```diff
@@ -82,4 +82,3 @@ def init_coco_format():
     # 为每个分区保存JSON文件
 with open(f"./annotations/instances_val2017.json", "w") as json_file:
     json.dump(coco_format, json_file, indent=4)
-
```
Comment (zhanghuiyao) [inline examples/finetune_car_detection/crejson.py:85]: 空行

File: examples/finetune_car_detection/voc2yolo.py [hunk 3]
```diff
@@ -43,7 +42,7 @@ def xml2txt(xml, labels, name_list, img_path):
         fil_name = root.find('filename').text[:-4]
         if not os.path.exists(labels):  # 如果路径不存在则创建
             os.mkdir(labels)
-        out = open(labels + './' + fil_name + '.txt', 'w+')
+        out = open(labels + '/' + fil_name + '.txt', 'w+')
```
Comment (zhanghuiyao) [inline examples/finetune_car_detection/voc2yolo.py:45]: 可以换成 os.path.join(path1, path2)的方式

**Review by zhanghuiyao — APPROVED** [review]

File: examples/finetune_car_detection/rename.py [hunk 4]
```diff
@@ -5,18 +5,17 @@
...
 
 for i, file in enumerate(files, start=start_num):
     # 构造新的文件名，使用zfill填充前导零
     new_filename = f"{i:06d}{extension}"
 
     # 使用os.rename重命名文件
     # 需根据自己的路径修改
-    os.chdir('D:/Desktop/mindspore/yolo_car_detection/bddua/bdd100k/images/train')
+    os.chdir('D:/Desktop/mindspore/yolo_car_detection/bddua/bdd100k/images/val')
```
Comment (CaitinZhao) [inline examples/finetune_car_detection/rename.py:16]: 这个搞成入参吧，代码里硬编码不好

Comment (yuedongli1) [reply examples/finetune_car_detection/rename.py]: 已修改为相对路径，并在readme中说明应在数据集的根目录下使用

**Review by CaitinZhao — APPROVED** [review]

**Review by zhanghuiyao — APPROVED** [review]

**Pull request merged**